docs/explanations/architecture-2.md

```diff
@@ -0,0 +1,331 @@
+---
+sidebar_label: Architecture - 2
+sidebar_position: 61
+title: Kubewarden architecture
+description: The Kubewarden architecture
+keywords: [kubewarden, kubernetes, architecture]
+doc-persona: [kubewarden-all]
+doc-type: [explanation]
+doc-topic: [architecture]
+---
+
+<!--
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/explanations/architecture"/>
+</head>
+-->
+
+Kubewarden is a Kubernetes policy engine.
+It uses policies written in a programming language of your choosing.
+This language must generate a WebAssembly binary for Kubewarden to use.
+
+## Something to clear up?
+
+What's the difference between
+
+- PolicyServer and policy-server.
+
+An up-front explanation of this type of usage would be useful. I think it
+occurs elsewhere in the docs, with other terms as well. Does it need
+clarification?
```

Comment:
As far as I can see, we use both term interchangeably. They usually refers to the policy engine running in the cluster. But I thinks it's a good idea to create some standard for this term.